samples/simple_app/app.py

```diff
@@ -61,34 +51,55 @@ async def get_webhook_details(domain_name: DomainName) -> WebhookData:
         webhook_secret_key="webhook-secret-key",
     )
 
+async def example_dependency():
+    return "example"
+
 
-async def product_created(payload: Payload, saleor_domain: DomainName):
+async def product_created(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product created!")
     print(payload)
 
 
-async def product_updated(payload: Payload, saleor_domain: DomainName):
+async def product_updated(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product updated!")
     print(payload)
 
 
-async def product_deleted(payload: Payload, saleor_domain: DomainName):
+async def product_deleted(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product deleted!")
     print(payload)
 
 
-webhook_handlers = WebhookHandlers(
-    product_created=product_created,
-    product_updated=product_updated,
-    product_deleted=product_deleted,
-)
-
-
 app = SaleorApp(
+    manifest=Manifest(
+        name="Sample Saleor App",
+        version="0.1.0",
+        about="Sample Saleor App seving as an example.",
+        data_privacy="",
+        data_privacy_url="http://172.17.0.1:5000/dataPrivacyUrl",
+        homepage_url="http://172.17.0.1:5000/homepageUrl",
+        support_url="http://172.17.0.1:5000/supportUrl",
+        id="saleor-simple-sample",
+        permissions=["MANAGE_PRODUCTS", "MANAGE_USERS"],
+        configuration_url=Manifest.url_for("configuration-form"),
```

Comment:
This is an example of a lazy loader, the manifest endpoint resolves those once the HTTP request is available.